nautobot/docs/release-notes/version-2.4.md

```diff
@@ -151,6 +151,26 @@ As Python 3.8 has reached end-of-life, Nautobot 2.4 requires a minimum of Python
 
 <!-- towncrier release notes start -->
 
+## v2.4.1 (2025-01-17)
+
+### Security in v2.4.1
+
+- [#6780](https://github.com/nautobot/nautobot/issues/6780) - Updated `Django` to `4.2.18` to address `CVE-2024-56374`.
+
+### Fixed in v2.4.1
+
+- [#6427](https://github.com/nautobot/nautobot/issues/6427) - Fixed a bug which allowed several wireless interface types to accept cables.
+- [#6489](https://github.com/nautobot/nautobot/issues/6489) - Fixed partial-match filters (such as "__ic" and "__isw") on fields that have restricted choices (Prefix.type, Interface.type, etc.) so that partial values are no longer rejected.
```

Comment:
```suggestion
- [#6489](https://github.com/nautobot/nautobot/issues/6489) - Fixed partial-match filters (such as `__ic` and `__isw`) on fields that have restricted choices (`Prefix.type`, `Interface.type`, etc.) so that partial values are no longer rejected.
```